client/multi-currency/enabled-currencies-list/test/index.js

```diff
@@ -226,7 +226,28 @@ describe( 'Multi Currency enabled currencies list', () => {
 		expect( modal ).toMatchSnapshot();
 	} );
 
+	test( 'Remove currency modal doesnt render when theres no dependency', () => {
+		getContainer();
+		expect(
+			screen.queryByRole( 'dialog', { name: /remove euro/i } )
+		).not.toBeInTheDocument();
+		fireEvent.click(
+			screen.getByRole( 'button', {
+				name: /remove euro as an enabled currency/i,
+			} )
+		);
+		const modal = screen.queryByRole( 'dialog', { name: /remove euro/i } );
+		expect( modal ).not.toBeInTheDocument();
+	} );
+
 	test( 'Remove currency modal renders correctly', () => {
+		global.multiCurrencyPaymentMethodsMap = {
```

Comment:
I wonder if is there another option rather than making that map global and then erasing it my assigning to undefined. Can't suggest a better way, but maybe it exists

Reply:
It's for mocking these lines to change the bindings:

https://github.com/Automattic/woocommerce-payments/blob/4931ef764fe324678fe136445a5150d1adba044a/includes/multi-currency/PaymentMethodsCompatibility.php#L154-L159

`global` keyword works like `window` on the browser, so I've used that for testing. Maybe `window` will also work, maybe it's more understandable than `global`. I'll try and change it.

Reply:
Ah, thanks for the explanation. That's just TIL, if it's no difference then you can leave it as it is :) Good from my side.

Reply:
Changed `global` to `window` in 57fb725